tests/test_widgets.py

```diff
@@ -8,22 +9,39 @@
 )
 
 
+def example_number(region_code: str) -> PhoneNumber:
+    number = phonenumbers.example_number(region_code=region_code)
+    e164 = phonenumbers.format_number(
+        numobj=number, num_format=phonenumbers.PhoneNumberFormat.E164
+    )
+    return PhoneNumber.from_string(phone_number=e164, region=region_code)
+
+
+EXAMPLE_NUMBERS = (
+    example_number(region_code="US"),
+    example_number(region_code="CA"),
+    example_number(region_code="GB"),
+    example_number(region_code="GG"),
+    example_number(region_code="PL"),
```

Comment:
Can pass the arguments as positional without the name. Up to you.

Reply:
Haha no worries. Either way works.
The main rationale for me is:
1. Supplying positional argument positionally. The method signature has been made to accept the argument positionally, there might be a logic in their order. It also helps identifying their nature (e.g. order matters)
2. Avoid repeating the argument name (e.g. `can_view(user=user)`).
3. Slightly better performance https://peps.python.org/pep-0570/#performance
4. It helps with refactoring and grepping through the code, with more uniform call patterns.